tests/lib/snaps/store/test-snapd-with-default-configure-core18/meta/hooks/configure

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# Snapshot information
```

Comment:
It seems like all configure and default-configure hooks bewetten test-snapd-with-default-configure{,-core18,-core20,-core22} are identical (?) - if so, I wonder if there is a way to avoid the duplicatoin? I guess using a symlink would not work as `snap pack`would just include the symlink and not the symlink target but maybe a comment that all the hooks should/need to be kept in sync? Should not really be a problem because this probably will never change but a small comment also does not hurt

Reply:
Very similar indeed, the only subtle difference is the "tag" that the hook appends to the gadget variable which is used to demonstrate the order in which they execute, and that only default-configure applies the gadget defaults when it exists.

https://github.com/snapcore/snapd/pull/12747/files#diff-74475a5f82f71eb925e94a155ecf8172fdbf9459427efb40ceb1705282326c4bR17
VS 
https://github.com/snapcore/snapd/pull/12747/files#diff-9202dc18cb33bf1246e85e4b91766e76e28890bfbaab9329193ccca4659fc009R17

I will add a comment